README.md

```diff
@@ -54,15 +54,19 @@ every 3.hours do # 1.minute 1.day 1.week 1.month 1.year is also supported
   command "/usr/bin/my_great_command"
 end
 
-every 1.day, :at => '4:30 am' do
+every 1.day, at: '4:30 am' do
   runner "MyModel.task_to_run_at_four_thirty_in_the_morning"
 end
 
+every 1.day, at: ['4:30 am', '6:00 pm'] do # Also you can run in 1 hour for every five minutes: (0..60).step(5).map { |minute| "4:#{minute.to_s.rjust(2, '0')} am" }
```

Comment:
The example in the comment I think is too specific and complex for the README. Would you remove it?

Reply:
Thanks. I understand your policy of the README and just removed.